clis/teliod/Cargo.toml

```diff
@@ -34,6 +34,13 @@ anyhow.workspace = true
 smart-default = "0.7.1"
 base64 = "0.22.1"
 dirs = "4.0.0"
+const_format = { version = "0.2.33", optional = true }
+rust-cgi = { version = "0.7.1", optional = true }
+arrayvec = "0.7.6"
```

Comment:
This doesn't seem to be used anywhere - please remove it.

Reply:
removed